x-pack/plugins/cases/public/components/custom_fields/draggable/index.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiDragDropContext,
+  EuiDroppable,
+  EuiDraggable,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiText,
+  EuiIcon,
+} from '@elastic/eui';
+
+import type { CustomFieldTypes } from '../../../../common/types/domain';
+import { builderMap } from '../builder';
+export interface ListOption {
+  content: string;
+  id: string;
+  type?: CustomFieldTypes;
+}
+
+export interface Props {
+  listValues: ListOption[];
+}
+
+const DraggableComponent: React.FC<Props> = (props) => {
```

Comment:
I wanted to keep it as a component not related to custom field so that I can reuse it for List options drag and drop in future. Linking it with `CustomFieldsList` and  `CustomFieldsConfiguration` will prevent from reusing

Reply:
I forgot about that. I was confused with `renderTypeLabel` which contains logic specific to custom fields.

Reply:
yeah, that's true 😄  I will try to separate the label render logic when reuse for list options in another PR. is that okay?

Reply:
Ofc!